core/src/components/content/test/basic/content.e2e.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('content: basic', () => {
+  test('should not have visual regressions', async ({ page }) => {
+    await page.goto(`/src/components/content/test/basic`);
```

Comment:
Do we need to keep the RTL tests here? (Same question for all the other test files)

Reply:
Similarly, do we need per-mode tests? Content only have a single `content.scss` file.

Reply:
Hmm, I think RTL should be kept in the `fixed` test since it affects the positions of things, but otherwise yeah we're just checking if the browser correctly right-aligns the text. I'll remove it from the others.

I'm also good with removing one of the modes since the only difference is the elements within the `ion-content`, which are obviously checked in their own tests.